libraries/AP_Proximity/AP_Proximity_LeddarVu8.cpp

```diff
@@ -0,0 +1,247 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "AP_Proximity_LeddarVu8.h"
+#include <AP_HAL/AP_HAL.h>
+#include <ctype.h>
+#include <GCS_MAVLink/GCS.h>
+#include <GCS_MAVLink/GCS_MAVLink.h>
+#if HAL_PROXIMITY_ENABLED && AP_PROXIMITY_LEDDARVU8_ENABLED
+
+// update the state of the sensor
+void AP_Proximity_LeddarVu8::update()
+{
+    if (!_initialized) {
+        request_distances();
+        // gcs().send_text(MAV_SEVERITY_WARNING,"requested distances");
+        _temp_boundary.reset();
+        _initialized = true;
+        last_distance_ms = AP_HAL::millis();
+    }
+
+    if ((AP_HAL::millis() - last_distance_ms) < LEDDARVU8_TIMEOUT_MS) {
+        // just initialized
+        set_status(AP_Proximity::Status::Good);
+    } else {
+        set_status(AP_Proximity::Status::NoData);
+        // gcs().send_text(MAV_SEVERITY_WARNING,"no data");
```

Comment:
Same thing as above. Ideally, we cant have any commented out code unless absolutely necessary, I'll ignore the rest of them